components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/listener/ClaimMetadataManagementAuditLogger.java

```diff
@@ -211,4 +204,22 @@ private String buildLocalClaimData(Event event) {
         }
         return data.toString();
     }
+
+    /**
+     * Get the initiator for audit logs.
+     *
+     * @param tenantDomain Tenant domain of the initiator.
+     * @return initiator.
+     */
+    private String getInitiator(String tenantDomain) {
+
+        if (LoggerUtils.isLogMaskingEnable) {
+            String username = CarbonContext.getThreadLocalCarbonContext().getUsername();
+            if (StringUtils.isNotBlank(username) && StringUtils.isNotBlank(tenantDomain)) {
+                return IdentityUtil.getInitiatorId(username, tenantDomain);
```

Comment:
in some places, you have checked if this is blank. maintain a consistent approach

Reply:
checked all the places where the getInitiatorId method has been invoked. `StringUtils.isNotBlank(username) && StringUtils.isNotBlank(tenantDomain)` this condition has been used in all those places.